doc/sphinx/source/esmvaldiag/observations.inc

```diff
@@ -1,7 +1,269 @@
 Contributing a CMORizing script for an observational dataset
 ************************************************************
 
+ESMValTool is designed to work with CF compliant data and 
+follows the CMOR tables from the CMIP data request, therefore 
+the observational datasets need to be CMORized for usage in ESMValTool.
+The following steps are necessary to prepare an observational
+data set for the use in ESMValTool.
+
+| `1. Check if your variable is CMOR standard`_
+| `2. Edit your configuration file`_
+| `3. Store your dataset in the right place`_
+| `4. Create a cmorizer for the dataset`_
+| `4.1 Cmorizer script written in python`_
+| `4.2 Cmorizer script written in NCL`_
+| `5. Run the cmorizing script`_
+| `6. Naming convention of the observational data files`_
+| `7. Test the cmorized dataset`_
+
+
+1. Check if your variable is CMOR standard
+==========================================
+
+Most variables are defined in the CMIP data request and can be found in the
+CMOR tables in the folder `/esmvalcore/cmor/tables/cmip5/Tables/
+<https://github.com/ESMValGroup/ESMValCore/tree/development/esmvalcore/cmor/tables/cmip5/Tables>`_,
+differentiated according to the ``MIP`` they belong to. The tables essentially
+follow the `PCMDI <https://github.com/PCMDI>`_ guidelines. If you find the
+variable in one of these tables, you can proceed to the next section.
+
+If your variable is not available in the standard CMOR tables,
+you need to write a custom CMOR table for the variable
+as outlined below and add it to `/esmvalcore/cmor/tables/custom/
+<https://github.com/ESMValGroup/ESMValCore/tree/development/esmvalcore/cmor/tables/custom>`_.
+
+To create a new custom CMOR table you need to follow these
+guidelines:
+
+- Provide the ``variable_entry``;
+- Provide the ``modeling_realm``;
+- Provide the variable attributes, but leave ``standard_name`` blank. Necessary
+  variable attributes are: ``units``, ``cell_methods``, ``cell_measures``,
+  ``long_name``, ``comment``.   
+- Provide some additional variable attributes. Necessary additional variable
+  attributes are: ``dimensions``, ``out_name``, ``type``. There are also
+  additional variable attributes that can be defined here (see the already
+  available cmorizers). 
+
+It is recommended to use an existing custom table as a template, to edit the
+content and save it as ``<short_name>.dat``.
+
+2. Edit your configuration file
+===============================
+
+Make sure that beside the paths to the model simulations and observations, also
+the path to raw observational data to be cmorized (``RAWOBS``) is present in
+your configuration file. 
+
+3. Store your dataset in the right place
+========================================
+
+The folder ``RAWOBS`` needs the subdirectories ``Tier1``, ``Tier2`` and
+``Tier3``. The different tiers describe the different levels of restrictions
+for downloading and using the observations. The unformatted (raw) observations
```

Comment:
Add an explanation of what those restrictions are?

Reply:
Done.